exec/java-exec/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -550,9 +558,9 @@ Pair<SqlNodeList, SqlNodeList> ParenthesizedCompoundIdentifierList() :
     SqlIdentifier id;
 }
 {
-    id = SimpleIdentifier() {list.add(id);}
+    id = CompoundIdentifier() {list.add(id);}
```

Comment:
1. Why this change is needed? 
2. Please updated java-doc to reflect new method behavior: `Parses a comma-separated list of simple identifiers`.